src/client/components/nutrients-table/nutrients-table.vue

```diff
@@ -128,13 +128,20 @@ export default {
 <style>
   .nutrients-table {
     min-width: 500px;
+    width: 100%;
   }
   .nutrients-table .md-table-head-container {
     height: 84px;
   }
   .nutrients-table .md-table-head-label {
     height: 56px;
   }
+  .nutrients-table__parcel-column {
```

Comment:
variants of the same element, so use a BEM-modifier? `nuntrients-table__column--parcel`?

Reply:
Agree, rewrote